docs/3.api/1.components/2.nuxt-page.md

```diff
@@ -15,6 +15,10 @@ links:
 It accepts same `name` and `route` props.
 ::
 
+::callout{color="amber" icon="i-ph-warning-duotone"}
+`<NuxtPage>` should be used instead of `<RouterView>` as the former takes addtional care on internal states. Otherwise, `useRoute()` may return incorrect paths.
```

Comment:
```suggestion
`<NuxtPage>` should be used instead of `<RouterView>` as the former takes additional care on internal states. Otherwise, `useRoute()` may return incorrect paths.
```

Reply:
Thanks for spotting the typo. A commit has been pushed to fix this.